src/style.css

```diff
@@ -1,3 +1,86 @@
+:root {
+  --color-hover: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+}
+
+.img-class {
+  overflow-clip-margin: content-box;
+  overflow: clip;
+}
+
+.img {
+  width: 40px;
+  height: 40px;
+  display: flex;
+}
+
+.header {
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  display: flex;
+}
+
+.nav {
+  height: 60px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.nav__item {
+  margin-right: 20px;
+  text-decoration: none;
+  list-style-type: none;
+  display: flex;
+}
```

Comment:
The `.nav__item` class is being styled with `display: flex;` but it doesn't seem to have any child elements to align. If there are no child elements to align, you can remove this property.